### PR TITLE
Prints results summary plainly - fixes #844

### LINKED ISSUE #844: Add option to print text summary

### PR DESCRIPTION
**Pull request by knod**
I wanted to add pixel comparisons test for this too, but it took me down too many rabbit holes and those tests will need discussion.

Also, not a huge deal, but for some reason the styles in the print dialogue are slightly different than the styles I get when I use Chrome's tools to emulate printing. Don't know how to troubleshoot those. [Edit: At least, it does when there's a summary there for an actual cliff.]

### CODE REVIEW

File: src/forms/output/Summary.js [hunk 1]
```diff
@@ -342,6 +342,8 @@ const Summary = function ({ client, openFeedback, snippets }) {
   }  // ends for each benefit
 
   // Ask for feedback
+  // Stays put when printing. A take-home hint
+  // that the tool is still a prototype
```
Comment (ethanbb) [inline src/forms/output/Summary.js:346]: Hm, I'm not seeing this in the print preview. Not sure why.

File: src/index.css [hunk 2]
```diff
@@ -478,3 +489,46 @@ ACCESSIBILITY - Also see any other focus styles
...
+  #root #App .ui.segment {
+    margin: 0;
+  }
+
+  @page {
+    size: portrait;
+  }
+
+  #App {
+    filter: grayscale(100%);
```
Comment (ethanbb) [inline src/index.css]: I feel like we don't necessarily have to make it black-and-white; people can print in black and white if they want to anyway. And some printers might sneakily still use color ink even if printing B&W content in color mode. But not a big deal.

**Review by ethanbb — COMMENTED** [review]
Neat! One more thing I noticed is that in the Chrome dialog, if you check "background graphics" (which is unchecked by default), the footer appears in a block of gray (well, green converted to gray). We might want to change the colors on that on our end, in case some browsers don't have that feature.

File: src/forms/output/Summary.js (hunk 1 above)
Comment (knod) [reply src/forms/output/Summary.js:346]: Weird, I'll double check later today.

File: src/index.css (hunk 2 above)
Comment (knod) [reply src/index.css]: If the setting is automatically set to print in black & white, then that's fine and we can get rid of a style. I'll do a bit of research on it to make sure.

File: src/forms/output/Summary.js (hunk 1 above)
Comment (knod) [reply src/forms/output/Summary.js:346]: I did it now instead, and I do see the 'let us know' line when there's a cliff. The button just doesn't have a background.

Comment (knod) [reply src/forms/output/Summary.js:346]: If you don't see it, could you post an image here of what you do see?

**Comment by knod** [comment]
@ethanbb : Oh, I think I misunderstood the note. Should the footer have no background color at all? What are your intentions, sir!?

File: src/forms/output/Summary.js (hunk 1 above)
Comment (ethanbb) [reply src/forms/output/Summary.js:346]: Oh, I didn't realize it was only supposed to show up if there's a cliff?

Comment (ethanbb) [reply src/forms/output/Summary.js:346]: Oh never mind, I wasn't reading closely and I thought this was the yellow bar asking for feedback above the tabs, for some reason.

**Comment by ethanbb** [comment]
@knod I was saying we should remove the background color from the footer with CSS if possible, rather than relying on Chrome doing it automatically.

**Comment by knod** [comment]
@ethanbb : Got it. Is that something that we can push off for a future PR? Is it acceptable as it is as a working feature?

**Comment by knod** [comment]
You're right, though - it looks like not all browsers give choices about backgrounds: https://answers.microsoft.com/en-us/windows/forum/apps_windows_10-msedge/microsft-edge-does-not-allow-users-to-print/f1631753-cfe6-44c0-be34-b1dd10ae9b46.

**Comment by knod** [comment]
Question still stands!

**Comment by knod** [comment]
Because of interesting issues with Travis, it looks like I have to close this and re-do the PR, which means we'll lose the comments. I'll try to remember to put another comment in here with the link to the new one.

**Pull request closed without merging**